src/convertToObject.js

```diff
@@ -6,7 +6,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const cssProperties = {};
+  const arrayOfItems = sourceString
+    .split(';') // ['ключ : свойство','ключ:свойство']
+    .filter((item) => item.includes(':')) // убираем  пустые элементы
+    .map((elem) => elem.split(':')); //[[ключ, свойство], [ключ, свойство]]
```

Comment:
There's no need to make a copy of the array before chaining other methods, as methods like `filter` and `map` do not mutate the original array. This adheres to the [CODE KNOWLEDGE] checklist item.